packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js

```diff
@@ -110,7 +117,43 @@ class ExportToCsvCommand extends Command {
             }
 
             stackAPIClient = this.getStackClient(managementAPIClient, stack);
+
+            if (branchUid) {
+              try {
+                const isBranchExist = await stackAPIClient.branch(branchUid).fetch();
+                if (isBranchExist && typeof isBranchExist === 'object') {
+                  stack.branch_uid = branchUid;
+                  stackAPIClient = this.getStackClient(managementAPIClient, stack);
+                }
+              } catch (error) {
+                switch (error.errorCode) {
+                  case 905: {
+                    cliux.error(
+                      'The branch you entered does not exist in the selected stack. Please enter a valid branch.',
+                    );
+                    process.exit(1);
+                  }
+                  case 412: {
```

Comment:
@akashbhosale97 Are we still managing this error handling through the response’s error codes?
This was supposed to be handled through the branch feature check utility right?

Reply:
changed it to use utility